static/styles/_base/_tables.scss

```diff
@@ -13,9 +15,8 @@ table {
   }
 
   th {
-    padding: 1rem;
-    height: 4rem; 
-    border-bottom: 1px solid $light-gray;
+    padding: 1rem !important;
```

Comment:
Ooooh, well I now see that we're not even using the npm module yet. So let's refactor this whenever we figure out how we're loading the built in styles.